src_python/fl_server.py

```diff
@@ -112,20 +116,20 @@ def send_model(self, client_socket):
         :return: None
         """
 
-        if self.ROUNDS == self.training_cycles:
+        if self.rounds == self.training_cycles:
             self.stop_flag = True
 
-        weights = self.GLOBAL_WEIGHTS
+        weights = self.global_weights
 
-        data = {"STOP_FLAG":self.stop_flag,"WEIGHTS":weights}
+        data = {"STOP_FLAG": self.stop_flag, "WEIGHTS": weights}
```

Comment:
```suggestion
        data = {'STOP_FLAG': self.stop_flag, 'WEIGHTS': weights}
```

Reply:
@thevindu-w I am wondering why not the Python code linter enforce such String literal format rules.